selfdrive/car/toyota/values.py

```diff
@@ -710,6 +711,7 @@ class CAR:
       b'\x028646F4203300\x00\x00\x00\x008646G26011A0\x00\x00\x00\x00',
       b'\x028646F4203400\x00\x00\x00\x008646G2601200\x00\x00\x00\x00',
       b'\x028646F4203500\x00\x00\x00\x008646G2601200\x00\x00\x00\x00',
+      b'\x028646F4203500\x00\x00\x00\x00!!!!!!!!!!!!!!!!',
```

Comment:
I show `\x028646F4203500\x00\x00\x00\x008646G2601200\x00\x00\x00\x00` in the logs for this fw for this vehicle, which matches the FW already in the list

Reply:
Can you share the specific drive where this FW was taken from?

Reply:
FROM Event 234d57e166f3444aba81026062494a66
Apr 3, 2020 12:02:50 PM UTCJSON (2.4 KB)
TAGS
branch:origin/release4
dirty:False
dongle_id: 00dc05336f1756d9
0.7.4-arne182
remote:https://github.com/arne182/openpilot
car doesn't match any fw: [<car.capnp:CarParams.CarFw builder (ecu = esp, fwVersion = "\x01F152642551\x00\x00\x00\x00\x00\x00", address = 1968, subAddress = 0)>, <car.capnp:CarParams.CarFw builder (ecu = engine, fwVersion = "\x018966342U4000\x00\x00\x00\x00", address = 1792, subAddress = 0)>, <car.capnp:CarParams.CarFw builder (ecu = eps, fwVersion = "8965B42170\x00\x00\x00\x00\x00\x00", address = 1953, subAddress = 0)>, <car.capnp:CarParams.CarFw builder (ecu = fwdRadar, fwVersion = "\x018821F3301200\x00\x00\x00\x00", address = 1872, subAddress = 15)>, <car.capnp:CarParams.CarFw builder (ecu = fwdCamera, fwVersion = "\x028646F4203500\x00\x00\x00\x00!!!!!!!!!!!!!!!!", address = 1872, subAddress = 109)>]

Reply:
Looks more like a sentry issue not rendering the characters correctly.